src/components/maps.js

```diff
@@ -0,0 +1,120 @@
+// src/pages/MapPage.jsx
+import React, { useEffect, useState } from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+import { MapContainer, TileLayer, Marker, Popup, useMap } from 'react-leaflet'
+import redMarker from "../images/marker/red_marker.png"
+import burger from "../images/category/burger.png"
+import pizza from "../images/category/pizza.png"
+import fried_chicken from "../images/category/fried_chicken.png"
+import steak from "../images/category/steak.png"
+import desserts from "../images/category/desserts.png"
+import { Icon } from "leaflet"
+import axios from 'axios'
+import 'leaflet/dist/leaflet.css'
+
+const MapPage = () => {
+    const { state } = useParams()
+    const [restaurants, setRestaurants] = useState([])
+    const navigate = useNavigate
```

Comment:
This should be `const navigate = useNavigate()`. but I am not sure how it is used in this code?

Reply:
That's a good catch! I was wondering why it cannot direct me to other page.